code/_compile_options.dm

```diff
@@ -142,3 +142,12 @@
 #warn In order to build, run BUILD.bat in the root directory.
 #warn Consider switching to VSCode editor instead, where you can press Ctrl+Shift+B to build.
 #endif
+
+/// Runs the game in "map test mode"
+/// Map test mode prevents common annoyances, such as rats from spawning and random light fixture breakage,
+/// so mappers can test important facets of their map (working powernet, atmos, good light coverage) without these interfering.
+// #define MAP_TEST
```

Comment:
does this really warrant being a uh, "compile flag", especially if it's gonna end up in user code?
I worry about approachablity to new devs

Reply:
Do you have an idea for an alternative? I do seriously think some of these are detrimental for mapper's ability to test their code

Reply:
maybe add it in the server tab?

Reply:
I'm not really sure it's an actual problem, but the dumb idiot in my head that wrote the first comment would be more comfortable with a global var.
Again I'm unsure if it's warrented, would like a second opinion.

Reply:
FWIW we already had a maptainer get owned by random light breakages when reviewing prs and at the very least EOB said it would be appreciated (and like, I'm making this because I would definitely use it) 

I guess a global var would work to if it's sufficiently easy to find

Reply:
I think what I would prefer is to make this a config option instead, so you can use the `server_overrides.txt` thing in order to leave `MAP_TEST` mode always on without always needing to futz with the compile options on staging/unstaging/committing.

Reply:
Config would work but I'm worried that's something a lot harder to find than a compile flag - we already point people to the compile options file to run unit tests locally, as an example

Reply:
It's comparable to teach someone to look at the config options imo, we already have stuff like that anyhow